Mathlib/Analysis/CStarAlgebra/Classes.lean

```diff
@@ -116,3 +116,33 @@ noncomputable instance [CStarAlgebra A] : CStarAlgebra Aᵐᵒᵖ where
 noncomputable instance [CommCStarAlgebra A] : CommCStarAlgebra Aᵐᵒᵖ where
 
 end MulOpposite
+
+section real_imaginary_part
+
+variable {A : Type*} [NonUnitalNormedRing A] [StarRing A] [NormedSpace ℂ A] [StarModule ℂ A]
+  [NormedStarGroup A]
+
+lemma realPart.norm_le (x : A) : ‖realPart x‖ ≤ ‖x‖ := by
+  have hmain : ‖x + star x‖ ≤ 2 * ‖x‖ := calc
```

Comment:
I am open to better suggestions regarding which file this lemma and the next should go.

Reply:
I just realize I didn't need `A` to be a ring at all, now there's a more reasonable option for placing these lemmas.